### PR TITLE
[WIP] Add basic .travis.yml

### PR DESCRIPTION
**Pull request by aleksanb**
Needs work, look here for help https://docs.travis-ci.com/user/database-setup/#PostgreSQL.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -166,7 +166,7 @@ group :development do
 
   # Simple command execution over SSH. Lightweight deployment tool.
   # Using our own version until https://github.com/mina-deploy/mina/pull/361 is merged.
-  gem 'mina', git: "git@github.com:samfundet/mina.git"
+  gem 'mina', git: "git://github.com/Samfundet/mina.git"
```
Comment (cristeahub) [inline Gemfile:169]: What implications in regards to security does this change have?

Comment (aleksanb) [inline Gemfile:169]: When using the git@github syntax one requires your ssh key to be added to the relevant github account, even though the repository is open source.
The git protocol simply clones down the git file at that location.

Comment (sigvef) [inline Gemfile:169]: Is it possible to tunnel the connection over https using e.g. https://github.com/Samfundet/mina.git? That way one might not have to compromise on security for convenience' sake.

**Comment by cristeahub** [comment]
What happened here? Was the prq un-WIP-ed (i.e. not a WIP any more), but the title not changed?

File: Gemfile (hunk 1 above)
Comment (aleksanb) [inline Gemfile:169]: According to https://git-scm.com/book/tr/v2/Git-on-the-Server-The-Protocols git:// seems to have no authentication or security whatsoever.
I thought it was a wrapper around https, but seems not so.
We should probably change it then?

**Pull request merged**